osquery/logger/plugins/aws_kinesis.h

```diff
@@ -47,6 +48,7 @@ class KinesisLogForwarder : public BufferedLogForwarder {
   std::shared_ptr<Aws::Kinesis::KinesisClient> client_{nullptr};
 
   FRIEND_TEST(KinesisTests, test_send);
+  int retry_count = FLAGS_aws_kinesis_failed_upload_retry_count;
```

Comment:
Also, place it before the `FRIEND_TEST`. And if you don't mind, add doc-blocks for the methods in this class! :D